presto-docs/src/main/sphinx/security/hive.rst

```diff
@@ -0,0 +1,241 @@
+===========================
+Hive Security Configuration
+===========================
+
+The default security configuration of the :doc:`/connector/hive` uses simple
+authentication.  In this mode, the Hive connector connects to a Hadoop cluster
+that has been configured to use ``simple`` authentication. All queries are
+executed as the user who runs the Presto process, regardless of the user who
+submits the query.
+
+The Hive connector provides addional security options to support Hadoop
+clusters that have been configured to use :ref:`Kerberos
+<hive-security-kerberos-support>` for greater security.
+
+When accessing :abbr:`HDFS (Hadoop Distributed File System)`, Presto can
+:ref:`impersonate<hive-security-impersonation>` the end user who is running the
+query. This can be used with HDFS permissions and :abbr:`ACLs (Access Control
+Lists)` to provide additional security for data.
+
+.. _hive-security-kerberos-support:
+
+Kerberos Support
+================
+
+In order to use the Hive connector with a Hadoop cluster that uses ``kerberos``
+authentication, you will need to configure the connector to work with two
+services on the Hadoop cluster:
+
+* The Hive metastore Thrift service
+* The Hadoop Distributed File System (HDFS)
+
+Access to these services by the Hive connector is configured in the properties
+file that contains the general Hive connector configuration.
+
+Hive Metastore Thrift Service
+-----------------------------
+
```

Comment:
I think I prefer Authentication to Configuration. Configuration sounds to me like we're going to change the configuration for the Hive Metastore Thrift Service rather than changing Presto's configuration.

I agree that the two sections should be parallel though.